src/graph.js

```diff
@@ -39,7 +39,7 @@ function categorizeStatus(s) {
     if (s == 'Backlog' || s == 'Ready for Dev') {
         return 'Backlog';
     }
-    if (s == 'In Progress' || s == 'In QA on feature branch' || s == 'In Code Review' || s == 'Resolved, on staging') {
+    if (s == 'In Progress' || s == 'In QA on feature branch' || s == 'In Code Review') {
```

Comment:
@APILLSBURY Thanks a lot! It's almost perfect, but there is a fragility between status categories and this ordered list to get them to appear in "workflow order" in the epic stats breakdown in the top-right-hand corner:

https://github.com/andrei-m/jira-graph/blob/db5aa34c0758dc813620ebb3a860b3bae203f812/src/graph.js#L498

This causes the "Resolved, On Staging" set to be left off of the epic stats breakdown where it was previously included in the "In Progress" category's set. The quickest fix is to just add this to the ordered array. A better fix is to convert the getBreakdownByStatus function to use an ordered Map instead of its current unordered Object.

Reply:
yeah you still need the ordered list. I also don't think that "On staging" should be a separate entry in the point breakdown. Try d575e5b and f19421f

Reply:
Thanks! I will take a look tomorrow